app/forms/supplementary/WarehouseIdentification.scala

```diff
@@ -21,25 +21,24 @@ import play.api.data.{Form, Forms}
 import play.api.libs.json.Json
 import utils.validators.FormFieldValidator.noLongerThan
 
-case class WarehouseIdentification(typeCode: Option[String], id: Option[String])
+case class WarehouseIdentification(id: Option[String])
 
 object WarehouseIdentification {
   implicit val format = Json.format[WarehouseIdentification]
 
   val formId = "IdentificationOfWarehouse"
 
-  val mapping = Forms.mapping(
-    "typeCode" -> optional(text().verifying("supplementary.warehouse.typeCode.error", noLongerThan(35))),
+  val mapping = Forms.mapping( 
     "identificationNumber" -> optional(
       text().verifying("supplementary.warehouse.identificationNumber.error", noLongerThan(35))
```

Comment:
So based on preparing data, this field must have minimum two characters, first is for ID and rest of the string is for type code. What do you think about adding validation for minimum 2 chars? We have it inside optional so it can't affect the optional field.